ajax/libs/babel-core/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "babel-core",
+  "description": "A compiler for writing next generation JavaScript",
+  "version": "5.6.15",
+  "author": "Sebastian McKenzie <sebmck@gmail.com>",
+  "homepage": "https://babeljs.io/",
+  "license": "MIT",
+  "repository": "babel/babel",
+  "keywords": [
+    "6to5",
+    "babel",
+    "classes",
+    "const",
+    "es6",
+    "harmony",
+    "let",
+    "modules",
+    "transpile",
+    "transpiler",
+    "var"
+  ],
+  "dependencies": {
+    "acorn-jsx": "^1.0.0",
+    "ast-types": "~0.7.0",
+    "babel-plugin-constant-folding": "^1.0.1",
+    "babel-plugin-dead-code-elimination": "^1.0.2",
+    "babel-plugin-eval": "^1.0.1",
+    "babel-plugin-inline-environment-variables": "^1.0.1",
+    "babel-plugin-jscript": "^1.0.1",
+    "babel-plugin-member-expression-literals": "^1.0.1",
+    "babel-plugin-property-literals": "^1.0.1",
+    "babel-plugin-proto-to-assign": "^1.0.3",
+    "babel-plugin-react-constant-elements": "^1.0.3",
+    "babel-plugin-react-display-name": "^1.0.3",
+    "babel-plugin-remove-console": "^1.0.1",
+    "babel-plugin-remove-debugger": "^1.0.1",
+    "babel-plugin-runtime": "^1.0.7",
+    "babel-plugin-undeclared-variables-check": "^1.0.2",
+    "babel-plugin-undefined-to-void": "^1.1.6",
+    "chalk": "^1.0.0",
+    "convert-source-map": "^1.1.0",
+    "core-js": "^0.9.0",
+    "debug": "^2.1.1",
+    "detect-indent": "^3.0.0",
+    "estraverse": "^4.0.0",
+    "esutils": "^2.0.0",
+    "fs-readdir-recursive": "^0.1.0",
+    "globals": "^6.4.0",
+    "home-or-tmp": "^1.0.0",
+    "is-integer": "^1.0.4",
+    "js-tokens": "1.0.1",
+    "line-numbers": "0.2.0",
+    "lodash": "^3.6.0",
+    "minimatch": "^2.0.3",
+    "output-file-sync": "^1.1.0",
+    "path-exists": "^1.0.0",
+    "path-is-absolute": "^1.0.0",
+    "private": "^0.1.6",
+    "regenerator": "0.8.31",
+    "regexpu": "^1.1.2",
+    "repeating": "^1.1.2",
+    "resolve": "^1.1.6",
+    "shebang-regex": "^1.0.0",
+    "slash": "^1.0.0",
+    "source-map": "^0.4.0",
+    "source-map-support": "^0.2.10",
+    "strip-json-comments": "^1.0.2",
+    "to-fast-properties": "^1.0.0",
+    "trim-right": "^1.0.0"
+  },
+  "npmName": "jquery",
```

Comment:
Is this supposed to be `babel-core`? I noticed that the `babel-core` version is from a couple of months ago (`v5.6.15`) even though it says `npm autoupdate enabled`  and I'm wondering if this is why.